docs/dashboards/types-of-components.md

```diff
@@ -9,11 +9,25 @@ Here is the overview of most important parameters, that can be defined for a com
 * `Cell` - id of the cell, in which the component should be placed
 * `Class` - CSS class
 * `Type` - the type of the component.
-* `Events` - object containing a pair of name of the event and callback function that should be called on a given event.
+* `Events` - object containing a pair of name of the event and callback function that should be called on a given event. The list of events can be found in the API Reference but the most common one is `mount`.
```

Comment:
It would be good to have a link to API here.

Reply:
I know but for now we don't have it 😄